scanpy/datasets/__init__.py

```diff
@@ -295,3 +295,107 @@ def pbmc3k_processed() -> AnnData:
         backup_url='https://raw.githubusercontent.com/chanzuckerberg/cellxgene/master/example-dataset/pbmc3k.h5ad',
     )
     return adata
+
+def SGE10x(sample_ID = 'V1_Breast_Cancer_Block_A_Section_1') -> AnnData:
+    """Processed Spatial Gene Expression data from 10x Genomics.
+    Database: https://support.10xgenomics.com/spatial-gene-expression/datasets
+    Parameters
+    ----------
+    sample_ID
+        The ID of data sample in 10x spatial data base.
+        It should be one of the followings:       
+        'V1_Breast_Cancer_Block_A_Section_1'
+        'V1_Breast_Cancer_Block_A_Section_2'
+        'V1_Human_Heart'
+        'V1_Human_Lymph_Node'
+        'V1_Mouse_Kidney'
+        'V1_Adult_Mouse_Brain'
+        'V1_Mouse_Brain_Sagittal_Posterior'
+        'V1_Mouse_Brain_Sagittal_Posterior_Section_2'
+        'V1_Mouse_Brain_Sagittal_Anterior'
+        'V1_Mouse_Brain_Sagittal_Anterior_Section_2'
+        By defaults, It is set to:
+        'V1_Breast_Cancer_Block_A_Section_1'
+
+    Returns
+    -------
+    Annotated data matrix.
+    """    
+    
+    
+    # setting filenames, tarfilenames, backup_rurls
+    # tarfilenames and backup_urls will be used for downloading data base
+    from urllib.parse import urljoin
+    files = {
```

Comment:
please use `dict(...)` syntax for dicts with names that are valid python identifiers (like those ones)

Reply:
Done.

Reply:
> To get everything formatted, just install black and do `black .` in the scanpy directory. Tip: black has instructions on how to run it automatically on changed files from VS Code or PyCharm.

Thanks for suggesting black. It's quite useful and I reformat the files. But to be honest I do not like it 100% :)
For a set of similar lines of codes I prefer to left spaces to make them more clear, like this:
aaaa = 
a       =
aaa   =
which black prevents.